README.md

```diff
@@ -163,12 +165,20 @@ Usage : upjs-plato [options] -d <output_dir> <input files>
       Specify a ESLint configuration file for ESLint linting
   -b, --babelConfig : String
       Specify a Babel configuration file for project parsing
+  -p, --projectRoot : String
+      Root directory of the project to analyze. Needed to run audit/outdated/depngn analysis
```

Comment:
@arielj What's the difference between doing one or the other? 

Current way:

```
upjs-plato -r -d report src
```

New way with this PR:

```
upjs-plato -r -d report -p src
```

Is `src` going to be used as root directory in both scenarios?

Reply:
-p is a path that has a `yarn.lock` or `package-lock.json` file for the tools that need that work (audit/outdated/depngn)

the last argument is the directory with specific js/ts files to analyse for the reports that run per file

In a Rails app, `-p` would be the Rails root path that has the yarn.lock file, and the last argument would be `app/javascripts` for example when using webpacker or `app/assets/javascript` when using sprockets

In a JS app, `-p` would be the root path for example, and the last argument will be the source of the app itself (usually `src`)

in a mono-repo for example those can be used to target specific apps inside the monorepo

maybe I can add some defaults for `-p`? like if `-p` is not defined we can fallback to the current dir

Reply:
I added more details in the readme, I made the `projectRoot` default to the current working directory so this option is not required and added a not about using it for monorepos